### DISCUSSION TITLE
why ubuntu21.04 CPU utilization rate can only reach 20%?

### DISCUSSION THREAD

**Discussion by swarfens** [madMAx43v3r/chia-plotter#590] [Q&A]
help me!

**Comment by swarfens** [comment]
I tried using ubuntu20.04, and the same thing was that my 5950xcpu couldn't run full

**Comment by archerallstars** [comment]
My 5950x runs around 70-85% with 16 threads plotting on Ubuntu 21.04. I believe that the CPU using percentage is based on all cores usage. However, when you look at per-core usage, you would see many cores running at full 100%. Therefore it's either the kernel that's set to work this way or the CPU policy.

If you're using Ubuntu 21.04, which has kernel 5.11, with 5950x, you're on the right track as there's a performance boost for Ryzen CPUs in 5.11. Please do not use 20.04 if you care about performance.

*I'm not using PBO as I'm running on a fan.
